common/src/main/java/org/broadleafcommerce/common/config/BroadleafCommonConfig.java

```diff
@@ -48,4 +65,64 @@ public JpaVendorAdapter blJpaVendorAdapter() {
         return vendorAdapter;
     }
 
+    @Bean
+    @Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+    public Compiler blClosureCompiler() {
+        return new Compiler();
+    }
+
```

Comment:
I'm not really sure why this is a prototype bean. It seems that we only need one implementation.